arviz/plots/dotplot.py

```diff
@@ -0,0 +1,142 @@
+import math
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dots(
+    values=None,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="grey",
+    intervalcolor="red",
+    markersize=None,
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    quantiles=None,
+    point_interval=None,
+    plot_kwargs=None,
+    interval_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+
+    """Plot distribution as dot plot.
+
+    If quantiles is specified then it will plot the quantile dot plot.
```

Comment:
and if not?

Reply:
Then it will set quantiles as the total number of points in data and then plot.